src/utils.js

```diff
@@ -0,0 +1,5 @@
+function randomIndex(min, max) {
```

Comment:
Use es6 arrow function here and `module.exports` in the same line:
```
exports.randomInt = (min, max) => {
 ...
}
```
Also I think `randomInt` is more appropriate name since we might use it for non-index related stuff.